introduction.md

```diff
@@ -0,0 +1 @@
+Hi to all, my name is Renee de Kruijf. I'am from Holland.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.